roles/nova-common/tasks/main.yml

```diff
@@ -73,6 +73,17 @@
   notify:
     - restart nova services
 
+- name: copy policy files to openstack-dashboard
+  template:
+    src: "etc/nova/policy.json"
+    dest: "/etc/openstack-dashboard/nova_policy.json"
+    owner: "{{ openstack_meta.apache[os].user }}"
+    group: "{{ openstack_meta.apache[os].group }}"
+    mode: 0640
```

Comment:
root should probably own this file, with apache as group ... otherwise you're giving apache permission to write to this file ... which is not a good thing.